.github/workflows/link-check-exclude.yml

```diff
@@ -0,0 +1,27 @@
+name: Link Check Exclude
+on: deployment
+
+jobs:
+  run:
+    name: Link Check Exclude Job
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Run the link check script
+      id: check
+      uses: "iterative/link-check.action@v0.3.0"
+      with:
+        source: "filesystem"
```

Comment:
tbh, hard to tell how is it different from the other two checks ... `reportUnusedPatterns` - does it mean that it checks only exclude links? or everything else + exclude links?

Can we run exclude links checks as part of the regular cron check? It should be enough I guess if it's not very simple and obvious to make it part of the PR check

Reply:
Basically, this was an adaptation to match the circleCI config and check exclusions on commit. 

`reportUnusedPatterns` is only useful when done on a full check, so I made the `reportUnusedPatterns=only` flag to enable `dryRun`, letting the script run on the whole site to check exclusions without actually pinging every link in the site.

So there's two checks on commit: A full-site sweep that checks exclusions without actually sending requests and a diff-only check that actually sends requests and checks if the links are alive.

As we discussed around the other comments, it seems we agree it'd be better to just do this on the daily check.